openapi/api.yml

```diff
@@ -121,14 +120,22 @@ paths:
           content:
             application/json:
               schema:
-                anyOf:
-                  - $ref: 'components.yml#/components/schemas/ValidationError'
-              example: { "detail": "Invalid file type",
-                         "status": 400,
-                         "title": "Bad Request",
-                         "type": "about:blank",
-                         "validation_errors": [ "Sheet \"Project_Dim\" column \"Project_ID\" should contain only unique values.",
-                                                "The following sheets contain no data: Empty Sheet." ] }
+                $ref: 'components.yml#/components/schemas/ValidationError'
+              example: {
+                "detail": "Workbook validation failed",
+                "validation_errors": {
+                    "TabErrors": {
+                        "Tab1": {
+                           "Section 1": [
+                              "Error Message 1",
+                              "Error Message 2"
+                           ]
+                        }
+                    }
+                },
+                "status": 440,
+                "title": "Bad Request"
```

Comment:
The format of this is different depending on if the errors are pre- or post-transformation.

Reply:
Added two examples